### PR TITLE
net-nntp/nzbget: bump to 18.0_pre1820

### PR DESCRIPTION
**Pull request by sbraz**
@gktrk

### CODE REVIEW

**Comment by gktrk** [comment]
@swegener

File: net-nntp/nzbget/nzbget-18.0_pre1820.ebuild [hunk 1]
```diff
@@ -21,18 +21,17 @@ IUSE="debug gnutls ncurses parcheck ssl test zlib"
...
 	virtual/pkgconfig"
 DOCS=( ChangeLog README nzbget.conf )
 
-PATCHES=(
-	"${FILESDIR}"/${P}_parcheck-tests-fix.patch
-)
-
 S=${WORKDIR}/${PN}-${PV/_pre*/-testing}
 
 pkg_pretend() {
```
Comment (SoapGentoo) [inline net-nntp/nzbget/nzbget-18.0_pre1820.ebuild]: please also perform these checks in `pkg_setup`. Far too many people rely on pkg_pretend to check package invariants, which it cannot guarantee. Preferably stick it in a function and call it in `pkg_pretend` and `pkg_setup`.

File: net-nntp/nzbget/nzbget-18.0_pre1820.ebuild [hunk 2]
```diff
@@ -21,18 +21,17 @@ IUSE="debug gnutls ncurses parcheck ssl test zlib"
 RDEPEND="dev-libs/libxml2
 	ncurses? ( sys-libs/ncurses:0 )
```
Comment (SoapGentoo) [inline net-nntp/nzbget/nzbget-18.0_pre1820.ebuild]: if it links against ncurses, you need `:0=`

**Comment by sbraz** [comment]
@gktrk @SoapGentoo This should be good to go now. I reported the test problem upstream.

**Comment by gktrk** [comment]
pinged swegener on IRC

**Comment by swegener** [comment]
Thanks Louis, merged!

**Pull request closed without merging**